examples/root_macros/generate_4body_decay_chain.C

```diff
@@ -0,0 +1,273 @@
+/*----------------------------------------------------------------------------
+ *
+ *   Copyright (C) 2016 - 2017 Antonio Augusto Alves Junior
+ *
+ *   This file is part of Hydra Data Analysis Framework.
+ *
+ *   Hydra is free software: you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation, either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   Hydra is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with Hydra.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *---------------------------------------------------------------------------*/
+
+/*
+ * generate_decay_chain.C
+ *
+ *  Created on: 25/03/2018
+ *      Author: Antonio Augusto Alves Junior
```

Comment:
Ciao Davide, thanks for this contribution. Please, exchange the `Author` (line 26) from mine to your name.